utils/rocvideodecode/roc_video_dec.cpp

```diff
@@ -388,7 +388,11 @@ int RocVideoDecoder::HandleVideoSequence(RocdecVideoFormat *p_video_format) {
     output_surface_info_.output_height = target_height_;
     output_surface_info_.output_pitch  = surface_stride_;
     output_surface_info_.output_vstride = (out_mem_type_ == OUT_SURFACE_MEM_DEV_INTERNAL) ? surface_vstride_ : videoDecodeCreateInfo.target_height;
-    output_surface_info_.disp_rect = disp_rect_;
+    if (!(crop_rect_.right && crop_rect_.bottom)) {
```

Comment:
Instead of doing this, we can set it to videoDecoderCreateInfo::disp_rect where it already checked for this condition